Resin/Analyzer.cs

```diff
@@ -2,18 +2,29 @@
 
 namespace Resin
 {
-    public class Analyzer
+    public class Analyzer:IAnalyzer
     {
+        public static readonly char[] DefaultTakoneSeparators =
+        {
+            ' ', '.', ',', ';', ':', '!', '"', '&', '?', '#', '*', '+', '|', '=', '-', '_', '@', '\'',
+            '<', '>', '�', '�', '�', '`', '(', ')', '[', ']', '{', '}', '/', '\\',
+            '\r', '\n', '\t'
+        };
+
         private readonly char[] _tokenSeparators;
 
+        public Analyzer() : this(DefaultTakoneSeparators)
```

Comment:
I mean, sure, this also makes the analyzer easier to reason about. It's state is constant after having been initialized, regardless of whether you used the default ctor and separatorns or carried in your own. Optional parameters is a quick way of enriching your API but they are confusing at times.